[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/adr/0007-store-academic-cycles-on-trainees.md

We decided to associate `trainees` with `academic_cycles`, storing two foreign
keys on the trainee table:

> * `start_academic_cycle_id` to record the academic cycle the trainee ’started'

Reviewer: ```suggestion
* `start_academic_cycle_id` to record the academic cycle the trainee ‘started’
```

  training in, and
* `end_academic_cycle_id` to record the academic cycle the trainee 'finished'
  in.

### Setting the associations

At various times during the trainee’s lifecycle the data can change requiring us

> to update the academic cycles e.g. we have a trainee with an course end date in

Reviewer: ```suggestion
to update the academic cycles, for example, we have a trainee with a course end date in
```

This association between trainees and academic years makes it easier and quicker
for us to query for all trainees starting in a particular academic year, which

> is especially useful for our ’start year' and 'end year' filters.

Reviewer: ```suggestion
is especially useful for our ‘start year’ and ‘end year’ filters.
```
